tests/integration/testlib/iprule.py

```diff
@@ -21,7 +21,7 @@
 import logging
 
 from libnmstate import iplib
-from . import cmd as libcmd
+from . import cmd as cmdlib
```

Comment:
That seems wrong, it should be `from . import cmdlib`. Thanks! :-)

Reply:
well spotted, thank you. fixed.